sql/updates/world/2016_01_XX_XX_world.sql

```diff
@@ -0,0 +1,26 @@
+-- THERE ARE A LOT OF EXCESS COMMENTS IN THIS FILE. They will be removed after code review.
+-- Add sniffed gossip texts to menu
+DELETE FROM `gossip_menu_option` where menu_id = 8095;
+INSERT INTO `gossip_menu_option` (`menu_id`, `id`, `option_icon`, `option_text`, `OptionBroadcastTextID`, `option_id`, `npc_option_npcflag`, `action_menu_id`, `action_poi_id`, `box_coded`, `box_money`, `box_text`, `BoxBroadcastTextID`) VALUES
+(8095,0,0, 'Send me to the Abyssal Shelf!', 16615,1,1,0,0,0,0,NULL,0), -- There are multiple entries for this text. Review would be appreciated by anyone who can see which text is correct.
+(8095,1,0,'Send me to Honor Point!',17936,1,1,0,0,0,0,NULL,0);
+
+-- Migrate NPC to use SmartAI
+UPDATE `creature_template` SET `AIName` = 'SmartAI', `ScriptName` = '' WHERE `entry` = 20235;
+
+-- Condition: Gossip menu ID 0 needs either quest ID 10163 or quest ID 10346 to be incomplete.
+-- Condition: Gossip menu ID 1 needs quest ID 10382 to be taken and not rewarded. I wasn't sure how to read the CPP condition.
```

Comment:
replacing both negative conditions? Looks like the line notes did something weird so it's not pointing me to the right place, sorry